modules/mono/build_scripts/build.godot.android.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e;
+set -o pipefail;
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GODOT_SOURCE_ROOT="$( cd "${DIR}/../../.." >/dev/null 2>&1 && pwd )"
+
+set -x;
+
+cd ${GODOT_SOURCE_ROOT};
+
+ANDROID_HOME=${ANDROID_HOME:-${HOME}/Android/Sdk}
+ANDROID_NDK_ROOT=${ANDROID_NDK_ROOT:-${ANDROID_HOME}/ndk-bundle}
+ANDROID_NDK_HOME=${ANDROID_NDK_HOME:-${ANDROID_NDK_ROOT}}
+
+MONO_VERSION=${MONO_VERSION:-5.20}
+
+export ANDROID_HOME ANDROID_NDK_ROOT ANDROID_NDK_HOME MONO_VERSION
+
+SCONS_JOBS=${SCONS_JOBS:-2}
+SCONS_MONO_STATIC=${SCONS_MONO_STATIC:-yes}
+SCONS_VERBOSE=${SCONS_VERBOSE:-yes}
+
+: ${MONO_SOURCE_ROOT:?Variable MONO_SOURCE_ROOT not set or empty}
+
+# The SCons script expects an absolute path
+MONO_SOURCE_ROOT="$( cd "${MONO_SOURCE_ROOT}" >/dev/null 2>&1 && pwd )"
+
+function scons_build {
```

Comment:
To comply with POSIX standards, shell functions should be declared without the word "function" and with parenthesis:

    scons_build() {